<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogomauimoottorisiirto</class>
 <widget class="QDialog" name="Dialogomauimoottorisiirto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonSuljeSiirto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sulje</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMista">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Valitse tili</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FI1234567891234567</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FI9988776655443322</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxMihin">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Valitse tili</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FI1234567891234567</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FI9988776655443322</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonPaivita">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Siirrä</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSiirrettavaKate">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSiirrettavaSumma">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditViestiVastaanottajalle">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Kirjoita viesti tähän</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMuunnettaTilinumero">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelVastaanottajanKate">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Siirrettävä summa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>tai kirjoita tilinumero</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTilinValintaLahettaja">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valitse miltä tililtä siirretään</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTilinValintaVastaanottaja">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Valitse mille tilille siirretään</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOtsikkoSiirto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>251</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/new/prefix1/Pictures/BankSimulLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelViesti">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Viesti vastaanottajalle</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>325</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>20</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Siirto</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
